epos4.sh

```diff
@@ -0,0 +1,77 @@
+package: EPOS4
+version: "%(tag_basename)s"
+tag: "v4.0.0-alice1"
+source: https://github.com/alisw/EPOS4.git
+requires:
+  - ROOT
+  - fastjet
+env:
+  EPOVSN: "4.0.0"
+  EPO: "${EPOS_ROOT}/epos4/"
+  LIBDIR: "${EPO}bin"
+  OPT: "./"
+  HTO: "./"
+  CHK: "./"
+---
+#!/bin/bash -ex
+
+export EPOVSN=${PKGVERSION}
+
+# The following two variables *must* have a trailing slash! EPOS installation
+# will make a mess otherwise.
+export EPO=$PWD/
+export LIBDIR=${EPO}bin
+export CC=gcc
+export CXX=g++
+export FC=gfortran
+export FASTSYS=${FASTJET}
+export COP=BASIC
+
+rsync -a --exclude='**/.git' --delete ${SOURCEDIR}/ .
+
+export LIBRARY_PATH="$LD_LIBRARY_PATH"
+cmake -B$LIBDIR
+make -C$LIBDIR -j8
+
+# "Install"
+INST_SUBROOT=$INSTALLROOT/epos4
+mkdir -p $INST_SUBROOT 
+rsync -a \
+      --exclude='**/CMakeModules' \
+      --exclude=CMakeLists.txt \
+      --exclude='**/.git' \
+      --exclude=*.h \
+      --exclude=*.c \
+      --exclude=*.cpp \
+      --exclude=*.f \
+      ./ $INST_SUBROOT/
+chmod u+x $INST_SUBROOT/scripts/epos      
+find $INSTALLROOT -type d -empty -exec rmdir '{}' \; > /dev/null 2>&1 || true
+[ -d "$INST_SUBROOT" ]
+
+# Modulefile
+MODULEDIR="$INSTALLROOT/etc/modulefiles"
+MODULEFILE="$MODULEDIR/$PKGNAME"
+mkdir -p "$MODULEDIR"
+cat > "$MODULEFILE" <<EoF
+#%Module1.0
+proc ModulesHelp { } {
+  global version
+  puts stderr "ALICE Modulefile for $PKGNAME $PKGVERSION-@@PKGREVISION@$PKGHASH@@"
+}
+set version $PKGVERSION-@@PKGREVISION@$PKGHASH@@
+module-whatis "ALICE Modulefile for $PKGNAME $PKGVERSION-@@PKGREVISION@$PKGHASH@@"
+# Dependencies
+module load BASE/1.0 ROOT/$ROOT_VERSION-$ROOT_REVISION fastjet/$FASTJET_VERSION-$FASTJET_REVISION
+# Our environment
+set EPOS_ROOT \$::env(BASEDIR)/$PKGNAME/\$version
+setenv EPOS_ROOT \$EPOS_ROOT
+setenv EPOVSN 4.0.0
+# Final slash is required by EPOS, please leave it be
+setenv EPO \$::env(EPOS_ROOT)/epos4/
+prepend-path PATH \$::env(EPO)bin
+setenv LIBDIR \$::env(EPO)bin
+setenv OPT ./
+setenv HTO ./
+setenv CHK ./
```

Comment:
Is this what you really need? These will be set at runtime when you load the module.

Reply:
It's in the tutorial also of the installation of EPOS4 standalone. They need to be set and that should be the default one, but the user can change it afterwards obviously.